ui/lib/core/addon/templates/components/replication-header.hbs

```diff
@@ -25,9 +25,21 @@
           <span class="sep">
             /
           </span>
-          {{#link-to "vault.cluster.replication-dr-promote"}}
-            Disaster Recovery
-          {{/link-to}}
+          {{#if isDisabled}}
+            {{#link-to "vault.cluster.replication.index"}}
+              Replication
+            {{/link-to}}
+          {{else}}
+            {{#if (eq pageType "manage")}}
+              {{#link-to "vault.cluster.replication-dr-promote.details"}}
```

Comment:
A little curious why this link wouldn't go back to general Replication dashboard (`vault.cluster.replication.index`) instead. Or at the very least, maybe naming the breadcrumbs `Disaster Recovery Details` and `Disaster Recovery Manage` to be more indicative of what it's doing.

Reply:
Good thoughts.  @noelledaley do you have any opinions here?

Reply:
Confirmed with design that we'll remove breadcrumbs if it's a DR Secondary.